### PR TITLE
fix #33587 hihbt.com

### LINKED ISSUE #33587: hihbt.com
### Issue URL (Ads)
[NSFW][https://hihbt.com/4487.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fhihbt.com%2F4487.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.0.13
Browser: | Firefox
Filters: | EasyList China,<br>AdGuard Annoyances,<br>AdGuard Simplified domain names,<br>AdGuard Base,<br>Malware Domains,<br>NoCoin Filter List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/33587

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -524,6 +524,8 @@ alluc.ee###resultlist > div#resultitems[-ext-has="> div"]~iframe:not([src])
 levelsex.com,fapjournal.com#?#.content-wrap > div > .related:has(> div > .recommended-logo)
 techgenix.com#?#ul.sidebar_widget > li:has(> div > .dfp_ad_pos)
 techgenix.com#?#ul.sidebar_widget > li:has(> div > .ezoic-ad)
+hihbt.com#?#.theiaStickySidebar > .widget_custom_html:has(> div.textwidget > p[style] > a[href*="baa.im/"])
```
Comment (BlazDT) [inline EnglishFilter/sections/css_extended.txt]: Move both to foreign too.

File: EnglishFilter/sections/foreign.txt [hunk 2]
```diff
@@ -792,6 +792,17 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
...
+||a304.top^$image,domain=hihbt.com
+||z2x5c8.com^$image,domain=hihbt.com
+hihbt.com###advert_widget
+hihbt.com##aside[id^="advert-"]
+hihbt.com##a[href*="18sapp.com"]
+hihbt.com##a[href*="twseb20.com"]
+hihbt.com##a[href*="ap996.co/"]
+hihbt.com##a[href*="baa.im/"]
+hihbt.com##a[href*="goen.win/"]
+hihbt.com##a[href*="66dage.com/"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: .single-content > p > a[target="_blank"]

File: EnglishFilter/sections/foreign.txt [hunk 3]
```diff
@@ -792,6 +792,17 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/33601
 lsj2017.com##.wrap > .top_box
 ||107.175.181.58^$domain=lsj2017.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/33587
+||a304.top^$image,domain=hihbt.com
+||z2x5c8.com^$image,domain=hihbt.com
+hihbt.com###advert_widget
+hihbt.com##aside[id^="advert-"]
+hihbt.com##a[href*="18sapp.com"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: .single-content > p > a[target="_blank"]

File: EnglishFilter/sections/foreign.txt [hunk 4]
```diff
@@ -792,6 +792,17 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
...
+hihbt.com##a[href*="twseb20.com"]
+hihbt.com##a[href*="ap996.co/"]
+hihbt.com##a[href*="baa.im/"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: .textwidget  > p a[target="_blank"][rel="noopener"]:not([href^="/"]). Please also exchange search parameter for ext-CSS rules.

File: EnglishFilter/sections/foreign.txt [hunk 5]
```diff
@@ -792,6 +792,17 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
...
+hihbt.com##a[href*="goen.win/"]
```
Comment (BlazDT) [inline EnglishFilter/sections/foreign.txt]: .textwidget  > p a[target="_blank"][rel="noopener"]:not([href^="/"]). Please also exchange search parameter for ext-CSS rules.

**Review by BlazDT — CHANGES_REQUESTED** [review]
Those CSS rules which has no comment were not reproduced.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**